application-analytics-ui/src/main/resources/Analytics/Code/Macros/VelocityMacros.xml

```diff
@@ -63,7 +63,11 @@
 #end
 
 #macro (analytics_createLinkElement $label $link)
-&lt;a href = $link&gt;$label&lt;/a&gt;
+#if($link)
+  &lt;a href = $link&gt;$label&lt;/a&gt;
```

Comment:
```suggestion
  &lt;a href=$link&gt;$label&lt;/a&gt;
```
Also, I'm not sure if the `$link` and `$label` are xml escaped when passed to `analytics_createLinkElement`. You should watch out for that. I also think it would be better to have them escaped inside the function so you wont need to duplicate the escaping every time u use this function